physionet-django/user/forms.py

```diff
@@ -158,8 +159,14 @@ def save(self):
                     name_components[1] = new_username
                     profile.photo.name = '/'.join(name_components)
                     profile.save()
+
+                if settings.STORAGE_TYPE == 'S3':
```

Comment:
As @grzegorz-wcislo mentioned, this wouldn't be necessary if we had a non-changing public ID for users. If we had that, we could name their profile photo after that, and would never need to change the file name.